src/main/java/com/example/onlinebookstore/dto/OrderItemResponseDto.java

```diff
@@ -0,0 +1,16 @@
+package com.example.onlinebookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
empty constructor is available by default if we do not have any other constructors for this class

Reply:
remove anno